cpp/include/nvgraph_gdf.h

```diff
@@ -132,6 +132,16 @@ gdf_error gdf_AnalyzeClustering_ratio_cut_nvgraph(gdf_graph* gdf_G,
                                                    gdf_column* clustering,
                                                    float* score);
 
+/**
+ * Wrapper function for Nvgraph extract subgraph by vertices
+ * @param gdf_G Pointer to GDF graph object, this is the input graph
+ * @param result Pointer to GDF graph object, this is the output must be a valid pointer
+ * @param vertices Pointer to GDF column object which contains the list of vertices to extract
+ * @return Error code
+ */
+gdf_error gdf_extract_subgraph_vertex_nvgraph(gdf_graph* gdf_G,
```

Comment:
So as I long as I understand based on comments,

gdf_graph* gdf_g and gdf_column* vertices are input parameters and gdf_graph* result is an output parameter. A function input parameter list having (input, output, input) is unorthodox. It seems like other functions in this file places input parameters before output parameters in the function input parameter lists, we should follow that convention. Also, if this function is not modifying gdf_G and vertices, we should enforce "const correctness".

So the input parameter should be
(const gdf_graph* gdf_G, const gdf_column* vertices, gdf_graph* result).

Reply:
Changed the order.

Reply:
Can't make this const gdf_graph* gdf_G.  The graph can be changed if there is no CSR representation.

Reply:
Changing vertices to const makes sense.  However, that would require changing the implementation in nvgraph.  It seems to me like this is more than we want to address now.  In fact, we probably should talk about how much we want to change nvgraph other than bug fixes.